### DISCUSSION TITLE
Using ANTs landmarkmatch to register the colin27 to a study template and it looks strange - help!

### DISCUSSION THREAD

**Discussion by arianayoum** [ANTsX/ANTs#1688] [Q&A] [answered]
Hello, I am trying to register the colin27 MNI brain to an older-adult study template. I would like to use specific markers to anchor this registration so I am using the landmarkmatch function. However, the end result looks strange with a little protruding bump on the registered brain that does not exist on the fixed nor moving brains. Attached is a picture of the registered brain with the protrusion. Any help is welcome - thank you!

![Screenshot 2024-02-29 at 3 50 06 PM](https://github.com/ANTsX/ANTs/assets/43019028/f59bca7d-6810-43d1-b85b-c83b4cf6cdbf)

**Comment by cookpa** [comment]
That script was written for hippocampus matching and uses tools that haven't been updated in a long time. Could you try with `antsRegistration`? Start with `antsRegistrationSyN.sh`, it will register the images and print a command you can modify. Then you can add a point set metric (put the image metric first).

As always, code and data enables us to give the most detailed advice.

**Reply by arianayoum** [reply]
Hi, I tried with the antsRegistrationSyN but there is still an abnormal protrusion. 
I know you said it hasn't been updated in a while, but is there a way to edit the landmarkmatch function so that it reduces the amount of warping/deformation? 

Attached are:
1) Warped - the output of the antsRegistrationSyN, of the registered colin27 brain to the older adult template brain
2) colin_skillstripped - the colin27 brain that's been skull-stripped with BET
3) ANTS70-74 - the older adult template brain

[Warped.nii.gz](https://github.com/ANTsX/ANTs/files/14473560/Warped.nii.gz)
[colin_skullstripped.nii.gz](https://github.com/ANTsX/ANTs/files/14473553/colin_skullstripped.nii.gz)
[ANTS70-74Years3T_brain_bias_corrected.nii.gz](https://github.com/ANTsX/ANTs/files/14473555/ANTS70-74Years3T_brain_bias_corrected.nii.gz)

**Reply by cookpa** [reply]
You can smooth the deformation field by replacing the smoothing term `Gauss[ 3,0 ]` to something like `Gauss[ 3,1 ]`

**Reply by cookpa** [reply, marked as answer]
The fixed image is cropped very tightly, which can cause boundary instability in registration. By padding the image

`ImageMath 3 fixed_pad.nii.gz PadImage ANTS70-74Years3T_brain_bias_corrected.nii.gz 20`

I was able to get better results with `antsRegistrationSyN.sh`.

**Reply by arianayoum** [reply]
Hello, thank you so much! It works well :) I didn't realize the cropping would affect it - makes sense, and I really appreciate your help!

**Comment by ntustison** [comment]
Also, related---general landmark matching for a variety of transforms is available in ANTsPy and ANTsR.  [Here's](https://gist.github.com/ntustison/12a656a5fc2f6f9c4494c88dc09c5621#file-b_2_ants_interfaces_landmark_registration-md) a couple of self-contained examples.
